src/presenter/board-presenter.js

```diff
@@ -0,0 +1,22 @@
+import BoardView from '../view/board-view.js';
+import TaskListView from '../view/task-list-view.js';
+import EditFormView from '../view/edit-form-view';
+import {render} from '../render.js';
+
+export default class BoardPresenter {
+  boardComponent = new BoardView();
+  taskListComponent = new TaskListView();
+
+  init = (boardContainer) => {
+    this.boardContainer = boardContainer;
+
+    render(this.boardComponent, this.boardContainer);
+    render(new EditFormView(), this.taskListComponent.getElement());
+    render(this.taskListComponent, this.boardComponent.getElement());
```

Comment:
Тут ты так же продолжаешь добавлять элементы, они у тебя пустые сейчас, но когда ты заменишь шаблон в `<li>` ниже по правкам, то все будет ок и ты сможешь вызывать этот метод в цикле, тем самым 3 раза отрисовав точки
```suggestion
    render(new PointView(), this.boardCompont.getElement());
```